packages/ember-htmlbars/tests/compat/handlebars_get_test.js

```diff
@@ -34,13 +34,13 @@ QUnit.module("ember-htmlbars: compat - Ember.Handlebars.get", {
 QUnit.test('it can lookup a path from the current context', function() {
   expect(1);
 
-  registry.register('helper:handlebars-get', function(path, options) {
+  registry.register('helper:handlebars-get', new HandlebarsCompatibleHelper(function(path, options) {
```

Comment:
Seems like this manual wrapping should not be needed.  If it is required does that mean we have broken exporting Handlebars style helpers from `app/helpers/foo-bar.js`?